pkgs/applications/video/gpu-screen-recorder/gpu-screen-recorder-gtk.nix

```diff
@@ -33,7 +41,8 @@ stdenv.mkDerivation rec {
     install -Dt $out/bin/ gpu-screen-recorder-gtk
     install -Dt $out/share/applications/ gpu-screen-recorder-gtk.desktop
 
-    wrapProgram $out/bin/gpu-screen-recorder-gtk --prefix PATH : ${lib.makeBinPath [ gpu-screen-recorder ]}
+    gappsWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ gpu-screen-recorder ]})
+    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libglvnd ]})
```

Comment:
Multiple screen still seems to be a bit buggy. But I got now some recording working.

Reply:
What multi screen issue are you referring to?

Reply:
It seems to suggest the wrong screen if I have two screens connected but the internal laptop one disabled. I worked around this by having both screens enabled at the same time.

Reply:
Do you mean that it tries to record the disabled screen or that only the name is incorrect in the gui?

Reply:
https://dl.thalheim.io/3XMIBoCRpZY1c60yDySYsQ/Screenshot_20231119_152456.png This shows DP-4 in the selection. But than gpu-screen-recorder identifys my screen as DP-1.

Reply:
Oh I see what the issue is. gpu-screen-recorder in the default.nix file refers to a very old commit (d6cf3b73549f48bb1a640e6359fc45ee776b7188). It should refer to the latest commit. But that should be fixed if you use the tar files mentioned in the other comment.

Reply:
fixed
the code that was posted in the issue had the version set to november 18 so I assumed the rev was up to date

Reply:
Removing /run/opengl-driver/lib from LD_LIBRARY_PATH causes gpu-screen-recorder-gtk to be unable to find libcuda. Do you have yours configured globally?